script/openqa-clone-job

```diff
@@ -147,8 +147,18 @@ sub usage($) {
 
 sub split_jobid {
     my ($url_string) = @_;
-    my $url          = Mojo::URL->new($url_string);
-    my $host_url     = $url->scheme . '://' . $url->host;
+    my $url = Mojo::URL->new($url_string);
+
+    # handle scheme being omitted
+    if (!$url->scheme) {
+        warn 'no schema specified, defaulting to http';
+        $url->scheme('http');
+    }
```

Comment:
that worked in before without a warning so I suggest we just do that the same, e.g. just `$url->scheme('http') unless $url->scheme;`

Reply:
changed